components/scream/src/physics/shoc/shoc_compute_l_inf_shoc_length_impl.hpp

```diff
@@ -32,6 +32,7 @@ ::compute_l_inf_shoc_length(
                                 [&] (const int k) -> Spack {
     return ekat::sqrt(tke(k))*dz_zt(k);
   }, denom);
```

Comment:
Another `team_barrier()` question: Here, I did not previously call the barrier, but I am computing `numer` and `denom` by performing a reduce over threads, then using those values to compute a scalar (`l_inf`). Will it be required here? Or is this over kill?